brownie/project/main.py

```diff
@@ -597,7 +598,9 @@ def from_brownie_mix(
 
     print(f"Downloading from {url}...")
     _stream_download(url, str(project_path.parent))
-    project_path.parent.joinpath(project_name + "-mix-master").rename(project_path)
+    project_path.parent.joinpath(project_name + "-mix-{}".format(default_branch)).rename(
+        project_path
+    )
```

Comment:
Style wise we generally use f-strings over `.format()` wherever possible and only use the latter when the string needs to be changed at a later point (as in line 56 and then 592). A simple string such as here should be written as f-string like `f"{project_name}-mix-{default_branch}"`. It helps with readability.

There are more strings in the PR further down that should also be changed from `.format()` to f-strings.